src/AppInstallerCLICore/Workflows/MSStoreInstallerHandler.cpp

```diff
@@ -249,7 +255,9 @@ namespace AppInstaller::CLI::Workflow
 
         // Policy check
         AppInstallManager installManager;
-        if (installManager.IsStoreBlockedByPolicyAsync(s_StoreClientName, s_StoreClientPublisher).get())
+        bool bypassStorePolicy = WI_IsFlagSet(context.GetFlags(), Execution::ContextFlag::BypassStorePolicy);
+
+        if (installManager.IsStoreBlockedByPolicyAsync(s_StoreClientName, s_StoreClientPublisher).get() && !bypassStorePolicy)

```

Comment:
Do we want to bypass GetIsAppAllowedToInstallAsync() check below? If not, we could use a better name to represent "we are only bypassing if the Store client is disabled, other checks continue to work as expected", like BypassIsStoreClientBlockedPolicyCheck?

Reply:
Changed the name!